README.md

```diff
@@ -53,10 +53,287 @@ swagger-mock-validator spec.json mock.json --outputDepth 5
 ```
 
 For more options on how to use the command run the command with the help flag.
+
+`swagger-mock-validator --help`
+
+```bash
+Usage: swagger-mock-validator [options] <swagger> <mock>
+
+Confirms the swagger spec and mock are compatible with each other.
+
+Basic Usage:
+The <swagger> and <mock> arguments should paths to the json files or urls to the json files.
+
+Supported Mock Formats:
+Pact
+
+Pact Broker:
+For providers using the pact broker the <mock> argument should be the url to the root of the
+pact broker and the provider name should be passed using the --provider option. This will
+automatically find the latest versions of the consumer pact file(s) uploaded to the broker for
+the specified provider name. The <swagger> argument should be the path or url to the swagger
+json file. Optionally, pass a --tag option alongside a --provider option to filter the retrieved
+pacts from the broker by Pact Broker version tags.
+
+If the pact broker has basic auth enabled, pass a --user option with username and password joined by a colon
+(i.e. THE_USERNAME:THE_PASSWORD) to access the pact broker resources.
+
+Options:
+  -V, --version                                   output the version number
+  -p, --provider [string]                         The name of the provider in the pact broker
+  -t, --tag [string]                              The tag to filter pacts retrieved from the pact broker
+  -u, --user [USERNAME:PASSWORD]                  The basic auth username and password to access the pact broker
+  -a, --analyticsUrl [string]                     The url to send analytics events to as a http post
+  -o, --outputDepth [integer]                     Specifies the number of times to recurse while formatting the output objects. This is useful in case of large complicated objects or schemas. (default: 4)
+  -A, --additionalPropertiesInResponse [boolean]  allow additional properties in response bodies, default true
```

Comment:
Should we default this to `false`? i.e. by default, this is likely to cause badness so a user must override?

Reply:
Yeah I think that is the preferred angle from our stand-point. We could put out a version as it is (as per existing behaviour), with the options, and then put out a major release, which changes the defaults?

essentially flipping `additionalPropertiesInResponse` to default to `false`